docker/docker.go

```diff
@@ -111,7 +111,7 @@ func main() {
 			*flTls = true
 			cert, err := tls.LoadX509KeyPair(*flCert, *flKey)
 			if err != nil {
-				log.Fatalf("Couldn't load X509 key pair: %s. Key encrypted?", err)
+				log.Fatalf("Couldn't load X509 key pair: %q. The key may be encrypted.", err)
```

Comment:
I don't think the sentence "The key may be encrypted" is correct because that implies it would wrong for it to be encrypted, but I think the loadX509KeyPair fails it would be because its _not_ encrypted.  Perhaps: "The key must be encrypted" instead?